src/components/Property/types.ts

```diff
@@ -1,6 +1,18 @@
 // These props are the same for all property types
-export interface ConcretePropertyBaseProps {
-  name: string;
-  description: string;
-  disabled: boolean;
+export interface PropertyProps {
```

Comment:
Is this name too generic?  I.e. is there a risk that we mix it up with the types in the top level types file?

Reply:
Any suggestions for another name?

Reply:
Nope, and feel free to leave it as is